client/src/components/ingredients/ImageScan.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, useRef, useState } from 'react';
+import Webcam from 'react-webcam';
+import { Box, Button, ButtonGroup, Flex, HStack, VStack, Text } from '@chakra-ui/react';
+
+const CustomWebcam: React.FC = () => {
+    const webcamRef = useRef<Webcam>(null); 
+    const [imgSrc, setImgSrc] = useState<string | null>(null);
+    const [message, setMessage] = useState<string | null>(null);
+
+    // capture photo
+    const capture = useCallback(() => {
+        if (webcamRef.current) {
+        const imageSrc = webcamRef.current.getScreenshot();
+        setImgSrc(imageSrc);
+        setMessage(null)
+        }
+    }, [webcamRef]);
+
+    // retake photo
+    const retake = () => {
+        setImgSrc(null);
+        setMessage(null);
+    };
+
+    // submit photo
+    const submit = () => {
+        if (imgSrc) {
+            setMessage('Sucessfully submitted');
+        }
```

Comment:
Please add some comments add here to let NA know that the image will be submitted at here!